solr/core/src/test-files/solr/configsets/cloud-hdfs/conf/solrconfig.xml

```diff
@@ -35,7 +35,9 @@
     <commitWithin>
       <softCommit>${solr.commitwithin.softcommit:true}</softCommit>
     </commitWithin>
-    <updateLog></updateLog>
+    <updateLog>
```

Comment:
@magibney this satisfies our usecase test?

Reply:
This (hdfs) is actually irrelevant to our use case. It's even not directly relevant to the PR (which doesn't really exercise explicit `solr.ulog.dir` configuration for hdfs), but it puts a hook in the config file to make such testing possible. This PR _does_ test regular local filesystem explicit `solr.ulog.dir`, which is our usecase (and the most common usecase generally, by far).